examples/azure-rules.yaml

```diff
@@ -17,6 +17,15 @@ groups:
         remediation: |
           Adjust the corosync tokem timeout as recommended on the Azure best practices. https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker 
         scored: true
+      - id: 1.1.1.runtime
+        description: "Corosync Token is set to 30000, check it runtime"
+        audit: 'corosync-cmapctl | grep "runtime.config.totem.token (u32) = " | sed "s/^.*= //"'
+        tests:
+          test_items:
+            - flag: 30000
+        remediation: |
+          Adjust the corosync tokem timeout as recommended on the Azure best practices. https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse-pacemaker
```

Comment:
maybe add also "restart.."? are we executing this test only if the on-disk comparison has succeeded (passed)?

Reply:
Sry, do you mean to restart the corosync service?